app/script/util/Environment.js

```diff
@@ -62,7 +62,8 @@ z.util.Environment = (() => {
   };
 
   const _getVersion = () => {
-    const [majorVersion] = window.platform.version.split('.');
+    const browserVersion = window.platform.version || [];
+    const [majorVersion] = browserVersion.split('.');
```

Comment:
Array doesn't have a method `split`. Should be defaulting to an empty string.

Also you could inline it
```
const [majorVersion] = (window.platform.version || '').split('.');
```